addons/website/models/website_visitor.py

```diff
@@ -183,19 +184,20 @@ def _get_visitor_from_request(self):
         elif visitor and visitor.partner_id:
             # Cookie associated to a Partner
             visitor = Visitor
+
+        if force_create and not visitor:
+            visitor = self._create_visitor()
+
         return visitor
 
     def _get_visitor_from_request_or_create(self):
```

Comment:
@JKE-be Can we adapt code in enterprise to remove this method?

Reply:
Or we keep this for backward compatibility?

Reply:
in theory, the release has been done.. but isnce it is a private method, and that it just make 2 days ... why not